readme.md

```diff
@@ -22,40 +23,43 @@ $ npm install terminal-image
 const terminalImage = require('terminal-image');
 
 (async () => {
-	console.log(await terminalImage.file('unicorn.jpg'));
+	console.log(await terminalImage('unicorn.jpg'));
 })();
 ```
 
+Optionally, you can specify the `height` and/or `width` to scale the image. To
+maintain aspect ratio, define only one of them.
 
-## API
-
-Supports PNG and JPEG images.
-
-### terminalImage.buffer(imageBuffer)
+```js
+const terminalImage = require('terminal-image');
 
-Returns a `Promise<string>` with the ansi escape codes to display the image.
+(async () => {
+	console.log(await terminalImage('unicorn.jpg', {width: 1024}));
+})();
+```
 
-#### imageBuffer
 
-Type: `Buffer`
+## API
 
-Buffer with the image.
+Supports PNG and JPEG images.
 
-### terminalImage.file(filePath)
+### terminalImage(fileBuffer, [options])
```

Comment:
`fileBuffer` is not correct when it can also be a filePath. Rather just call it `input`.

Reply:
Nice name! I was not able to find a good one :+1: